### PR TITLE
fixed short interest and low float commands list error

### PR DESCRIPTION
**Pull request by CosminIvanov**
Also made input support arrow keys

Maybe we can do something smarter later but this is a #quickfix for it to work at this moment.

### CODE REVIEW

File: gamestonk_terminal.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 import res_menu as rm
 import config_terminal as cfg
 from alpha_vantage.timeseries import TimeSeries
+import readline
```
Comment (shadycuz) [inline gamestonk_terminal.py:16]: is this needed? It doesn't seem like you are using it in this file?

Comment (shadycuz) [reply gamestonk_terminal.py:16]: I'm wrong it is needed

**Review by shadycuz — APPROVED** [review]

**Comment by aia** [comment]
Parsing logic is wrong there.

Instead of looking for TDs
```

    a_high_short_interested_stocks = re.sub('<!--.*?//-->','', text_soup_high_short_interested_stocks.find_all('td')[3].text, flags=re.DOTALL).split('\n')[2:]
    a_high_short_interested_stocks = re.sub('<!--.*?//-->','', text_soup_high_short_interested_stocks.find_all('td')[2].text, flags=re.DOTALL).split('\n')[2:]
    a_high_short_interested_stocks[0] = a_high_short_interested_stocks[0].replace('TickerCompanyExchangeShortIntFloatOutstdIndustry','')

    l_stock_info = list()
    for elem in a_high_short_interested_stocks:
        if elem is '':
            continue
```

You want to be looking for lines first (TRs):
```
    stock_list_tr = text_soup_high_short_interested_stocks.find_all("tr")

    shorted_stock_data = list()
    for a_stock in stock_list_tr:
        a_stock_txt = a_stock.text

        if a_stock_txt == "":
            continue
```

**Comment by aia** [comment]
readline will break Windows btw.

**Comment by shadycuz** [comment]
> readline will break Windows btw.

Pystan isn't supported on Windows. I don't think we will only support docker on windows in the future.

**Comment by aia** [comment]
> 
> 
> > readline will break Windows btw.
> 
> Pystan isn't supported on Windows. I don't think we will only support docker on windows in the future.

Windows is working fine:

```
(gt) c:\Users\av\git\gt\GamestonkTerminal>pip list | grep pystan
pystan                        2.19.1.1
```

There is absolutely no reason not to support windows and docker is not as solution as it does not interact with browsers and UI.

Readline is completely irrelevant on the other hand.

**Comment by shadycuz** [comment]
> Windows is working fine:
> 
> ```
> (gt) c:\Users\av\git\gt\GamestonkTerminal>pip list | grep pystan
> pystan                        2.19.1.1
> ```
> 
> There is absolutely no reason not to support windows and docker is not as solution as it does not interact with browsers and UI.
> 
> Readline is completely irrelevant on the other hand.

I guess I should have said "officially" [supported](https://pystan.readthedocs.io/en/latest/installation.html). If you google you will see most people can't get it installed and those that do have to use conda. I have tried every version between `2.14` and `2.19.1.1` but could not get a build to pass in [actions](https://github.com/DidierRLopes/GamestonkTerminal/actions/runs/616873331). If you can reliliably get `pystan` to install on a fresh version of windows that would be great, please submit a PR or open an issue with the steps.

**Comment by aia** [comment]
> 
> 
> > Windows is working fine:
> > ```
> > (gt) c:\Users\av\git\gt\GamestonkTerminal>pip list | grep pystan
> > pystan                        2.19.1.1
> > ```
> > 
> > 
> > There is absolutely no reason not to support windows and docker is not as solution as it does not interact with browsers and UI.
> > Readline is completely irrelevant on the other hand.
> 
> I guess I should have said "officially" [supported](https://pystan.readthedocs.io/en/latest/installation.html). If you google you will see most people can't get it installed and those that do have to use conda. I have tried every version between `2.14` and `2.19.1.1` but could not get a build to pass in [actions](https://github.com/DidierRLopes/GamestonkTerminal/actions/runs/616873331). If you can reliliably get `pystan` to install on a fresh version of windows that would be great, please submit a PR or open an issue with the steps.
```
conda install -c conda-forge fbprophet
```
Works very reliably for me. Anaconda on Windows is the only way to Python because of the very sketchy access to compilers and libraries that you have on Windows.

Pystan, fbprophet are just a few features that can be optional. Not supporting Gamestonk Terminal on Windows just because a couple of features don't work is very narrow-minded. Launching fbprophet in a Docker form Gamestonk Terminal running on Windows is a much better product design.

**Comment by DidierRLopes** [comment]
I had a chat with @aia and I believe he has fixed some web scraping bugs that I had where I was looking for cells (td) instead of rows (tr) in the javascript code. I believe this is one of them.

Regarding the readline, does the module pyreadline works for all?
https://stackoverflow.com/questions/51157443/pythons-readline-module-not-available-for-windows

**Comment by aia** [comment]
I tried pyreadline to get tab-completion working on Windows, but haven't had much luck. Terminal does not recognize TAB.

**Comment by aia** [comment]
This should be addressed now.

**Comment by DidierRLopes** [comment]
Hey @CosminIvanov ,

This should now have been fixed by @aia's PR. He improved the way of web scraping overall.

I'll have a look regarding the readline, to see if there's something out there that can be used, and works on all OS.

I'll close this PR now, thanks!

**Pull request closed without merging**